checks/run-staticAnalysis.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+# Script to run static code analysis
+
+#set strict pipefail option
+#set -eo pipefail
+
+# A path to the root folder of ls1 mardyn can be set via an argument; if not, the parent folder of "checks" is used
+if [ $# -eq 1 ]
+then
+  rootFolder=$1
+else
+  rootFolder=$PWD/..
+fi
+
+echo "Running in $rootFolder"
+
+warnings=""
+
+codeFiles=$( find $rootFolder/src -name "*.h" -or -name "*.cpp" -printf "%p " )
+
+# Similar to check_format of MegaMol repo
+for file in $codeFiles; do
+
+  # Check if using namespace std is used
+  if grep -q "using namespace std;" "$file"; then
+    echo "\"using namespace std;\" was used in $file"
+    warnings+="Do not use \"using namespace std;\"\n"
+  fi
+
+  # Check if using Log::global_log is used
+  if grep -q "using namespace std;" "$file"; then
+    echo "\"using Log::global_log;\" was used in $file"
+    warnings+="Do not use \"using Log::global_log;\"\n"
+  fi
+
+  # Check if file is UTF-8 (or ASCII)
+  encoding=$(file -b --mime-encoding "$file")
+  if [[ $encoding != "us-ascii" && $encoding != "utf-8" ]]; then
+    # Fix
+    #tmp_file=$(mktemp)
+    #iconv -f "$encoding" -t utf-8 -o "$tmp_file" "$file"
+    #mv -f "$tmp_file" "$file"
```

Comment:
Either we print the commands to fix this to the output or we remove these commented-out lines

Reply:
I have added the fix to the output. But somehow the CI does not seem to run now. Maybe a problem of GitHub or some settings?